src/lmic/radio_sx127x.c

```diff
@@ -31,7 +31,7 @@
 #define LMIC_DR_LEGACY 0
 
 #include "lmic.h"
-
+#if (CFG_sx1272_radio || CFG_sx1276_radio)
```

Comment:
please use `defined(` on both parts, for consistency with files above. 

Or, to minimize typing, we could introduce derived variables `..._sx127x_radio` and `..._sx126x_radio`, that are always defined, and either zero or non-zero. But I think that can be done later. (The `...` should be something other than `CFG` because `CFG` is used for user configuration `#defines`. There are examples elsewhere.)  However, I think it's better to just go for consistency and do a cleanup pass separately. You've waited long enough.